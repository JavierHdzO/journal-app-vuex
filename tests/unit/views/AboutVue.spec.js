
import { shallowMount } from "@vue/test-utils";
import AboutVue from '@/views/AboutVue.vue'

describe('AboutVue Component', () => {


    test('should match snapshot ', () =>{
        const wrapper = shallowMount(AboutVue);

        expect( wrapper.html()).toMatchSnapshot() 
    })
})
