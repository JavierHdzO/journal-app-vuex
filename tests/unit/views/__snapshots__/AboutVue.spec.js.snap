// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutVue Component should match snapshot  1`] = `<h1>About</h1>`;
